src/Data/TreeDiff/QuickCheck.hs

```diff
@@ -3,12 +3,18 @@ module Data.TreeDiff.QuickCheck (
     ediffEq,
     ) where
 
-import Data.TreeDiff
-import System.Console.ANSI (SGR (Reset), setSGRCode)
-import Test.QuickCheck     (Property, counterexample)
+import qualified Data.Text.Lazy                as TL
+import           Data.TreeDiff
+import           Prettyprinter
+                 (defaultLayoutOptions, layoutPretty)
+import           Prettyprinter.Render.Terminal (renderLazy)
+import           System.Console.ANSI           (SGR (Reset), setSGRCode)
+import           Test.QuickCheck               (Property, counterexample)
 
 -- | A variant of '===', which outputs a diff when values are inequal.
 ediffEq :: (Eq a, ToExpr a) => a -> a -> Property
 ediffEq x y = counterexample
-    (setSGRCode [Reset] ++ show (ansiWlEditExpr $ ediff x y))
+    (setSGRCode [Reset] ++ render (ansiWlEditExpr $ ediff x y))
     (x == y)
+    where
+        render = TL.unpack . renderLazy . layoutPretty defaultLayoutOptions
```

Comment:
AFAICT, this is the only relevant change, properties are not coloured with `prettyprinter`. If so, this is the only relevant change.

Reply:
Yes, #91 includes _only_ this change, which is why I linked to it in https://github.com/haskellari/tree-diff/pull/77#issuecomment-2498966539.